python/ray/client_builder.py

```diff
@@ -142,54 +142,12 @@ def _init_args(self, **kwargs) -> "ClientBuilder":
 
 
 class _LocalClientBuilder(ClientBuilder):
```

Comment:
why not remove this as well?

BTW, will this break backward compatibility with `ray.client().connect()`?

Reply:
Yeah, _LocalClientBuilder is the default for ray.client().connect()